RegisterController.js

```diff
@@ -0,0 +1,55 @@
+/*!
+ * Copyright (c) 2018 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+import {AccountService} from './AccountService.js';
+import pDebounce from 'p-debounce';
+
+export class RegisterController {
+  constructor({debounceExists = 500, accountServiceConfig = {}} = {}) {
+    this.state = {
+      registering: false,
+      checkingExistence: false,
+      email: null,
+      // phoneNumber: null
```

Comment:
What is going on with `phoneNumber`?

Reply:
I made an issue (#3) for it. It can be left alone.